website/common/script/ops/feed.js

```diff
@@ -38,6 +38,8 @@ function evolve (user, pet, req) {
 export default function feed (user, req = {}, analytics) {
   let pet = get(req, 'params.pet');
   const foodK = get(req, 'params.food');
+  let amount = get(req.query, 'amount', 1);
```

Comment:
this needs to be converted to a number because query parameters are strings, `let amount = Number(get(req.query, 'amount', 1));`. Should fix the failing test

Reply:
Thank you, Paglias for helping me out! I am learning!